File: client/src/app/_dtos/admin/jdDto.ts
export interface IJDDto {
     id: number;
     orderItemId: number;
     orderId: number;
     orderNo: number;
     orderDate: Date;
     customerName: string;
     categoryName: string;
     jobDescInBrief: string;
     qualificationDesired: string;
     expDesiredMin: number;
     expDesiredMax: number;
     minAge: number;
     maxAge: number;
}

File: api/DTOs/Admin/JDDto.cs
namespace api.DTOs.Admin
{
    public class JDDto
    {
        public int Id { get; set; }
        public string CompanyName { get; set; }
        public string CategoryName { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        public int OrderItemId { get; set; }    //foreign key
        public string JobDescInBrief { get; set; }
        public string QualificationDesired { get; set; }
        public int ExpDesiredMin { get; set; }
        public int ExpDesiredMax { get; set; }
        public int MinAge { get; set; }
        public int MaxAge { get; set; }
    }
}